Build Failure — Instructions Incomplete?

I want to send a PR or two, but that's a non-starter until I can get MassTransit to build.

### Is this a bug report?

Yes.

### Can you also reproduce the problem with the latest version?

Yes.

### Environment

1. Operating system: Windows 10
2. Visual Studio version: 2017 and 2019
3. Dotnet version: Framework 4.7.2, Core 2.2.202

### Steps to Reproduce

Follow steps in [Building from Source](https://github.com/MassTransit/MassTransit#building-from-source):

1. Clone the source down to your machine.
   Done.  Repo starts in the `develop` branch.
1. Run `build.bat`
    **Error:** There is no such file.  so...
   Run `build.ps1`
    **Error:** Build fails with errors.

Note that no build requirements are listed.

### Expected Behavior

MassTransit builds without error.

### Actual Behavior

Build fails with 1 warning, 35 errors.

```
> .\build.bat
.\build.bat : The term '.\build.bat' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ .\build.bat
+ ~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (.\build.bat:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```
```
> .\build.ps1
Preparing to run build script...
Running build script...

----------------------------------------
Setup
----------------------------------------

========================================
Clean
========================================

========================================
Restore-NuGet
========================================
  Restore completed in 191.46 ms for D:\Projects\Others\MassTransit\src\MassTransit.AzureServiceBusTransport.Tests\MassTransit.AzureServiceBusTransport.Tests.csproj.
  Restore completed in 191.45 ms for D:\Projects\Others\MassTransit\src\MassTransit.AzureServiceBusTransport\MassTransit.AzureServiceBusTransport.csproj.
  Restore completed in 197.32 ms for D:\Projects\Others\MassTransit\src\MassTransit.AutomatonymousIntegration.Tests\MassTransit.AutomatonymousIntegration.Tests.csproj.
  Restore completed in 5.4 ms for D:\Projects\Others\MassTransit\src\MassTransit.Futures\MassTransit.Futures.csproj.
  Restore completed in 195.44 ms for D:\Projects\Others\MassTransit\src\MassTransit.ApplicationInsights.Tests\MassTransit.ApplicationInsights.Tests.csproj.
  Restore completed in 209.89 ms for D:\Projects\Others\MassTransit\src\Containers\MassTransit.WindsorIntegration\MassTransit.WindsorIntegration.csproj.
  Restore completed in 210.11 ms for D:\Projects\Others\MassTransit\src\Containers\MassTransit.Automatonymous.StructureMapIntegration\MassTransit.Automatonymous.StructureMapIntegration.csproj.
  Restore completed in 210.54 ms for D:\Projects\Others\MassTransit\src\Loggers\MassTransit.NLogIntegration\MassTransit.NLogIntegration.csproj.
  Restore completed in 210.58 ms for D:\Projects\Others\MassTransit\src\MassTransit.AutomatonymousIntegration\MassTransit.AutomatonymousIntegration.csproj.
  Restore completed in 210.19 ms for D:\Projects\Others\MassTransit\src\Containers\MassTransit.LamarIntegration\MassTransit.LamarIntegration.csproj.
  Restore completed in 210.64 ms for D:\Projects\Others\MassTransit\src\Containers\MassTransit.Automatonymous.AutofacIntegration\MassTransit.Automatonymous.AutofacIntegration.csproj.
  Restore completed in 210.62 ms for D:\Projects\Others\MassTransit\src\Containers\MassTransit.Automatonymous.SimpleInjectorIntegration\MassTransit.Automatonymous.SimpleInjectorIntegration.csproj.
  Restore completed in 210.19 ms for D:\Projects\Others\MassTransit\src\Loggers\MassTransit.ExtensionsLoggingIntegration\MassTransit.ExtensionsLoggingIntegration.csproj.
  Restore completed in 210.92 ms for D:\Projects\Others\MassTransit\src\Containers\MassTransit.AutofacIntegration\MassTransit.AutofacIntegration.csproj.
  Restore completed in 210.88 ms for D:\Projects\Others\MassTransit\src\MassTransit.ApplicationInsights\MassTransit.ApplicationInsights.csproj.
  Restore completed in 210.96 ms for D:\Projects\Others\MassTransit\src\Containers\MassTransit.AspNetCoreIntegration\MassTransit.AspNetCoreIntegration.csproj.
  Restore completed in 210.6 ms for D:\Projects\Others\MassTransit\src\Containers\MassTransit.Automatonymous.LamarIntegration\MassTransit.Automatonymous.LamarIntegration.csproj.
  Restore completed in 210.84 ms for D:\Projects\Others\MassTransit\src\Loggers\MassTransit.SerilogIntegration\MassTransit.SerilogIntegration.csproj.
  Restore completed in 195.07 ms for D:\Projects\Others\MassTransit\src\MassTransit.Azure.ServiceBus.Core.Tests\MassTransit.Azure.ServiceBus.Core.Tests.csproj.
  Restore completed in 211.11 ms for D:\Projects\Others\MassTransit\src\Containers\MassTransit.SimpleInjectorIntegration\MassTransit.SimpleInjectorIntegration.csproj.
  Restore completed in 211.25 ms for D:\Projects\Others\MassTransit\src\Containers\MassTransit.NinjectIntegration\MassTransit.NinjectIntegration.csproj.
  Restore completed in 211.3 ms for D:\Projects\Others\MassTransit\src\MassTransit.AmazonSqsTransport\MassTransit.AmazonSqsTransport.csproj.
  Restore completed in 211.08 ms for D:\Projects\Others\MassTransit\src\Containers\MassTransit.ExtensionsDependencyInjectionIntegration\MassTransit.ExtensionsDependencyInjectionIntegration.csproj.
  Restore completed in 211.51 ms for D:\Projects\Others\MassTransit\src\Containers\MassTransit.StructureMapIntegration\MassTransit.StructureMapIntegration.csproj.
  Restore completed in 211.2 ms for D:\Projects\Others\MassTransit\src\Containers\MassTransit.Automatonymous.ExtensionsDependencyInjectionIntegration\MassTransit.Automatonymous.ExtensionsDependencyInjectionIntegration.csproj.
  Restore completed in 211.46 ms for D:\Projects\Others\MassTransit\src\MassTransit.ActiveMqTransport\MassTransit.ActiveMqTransport.csproj.
  Restore completed in 211.93 ms for D:\Projects\Others\MassTransit\src\Containers\MassTransit.Automatonymous.WindsorIntegration\MassTransit.Automatonymous.WindsorIntegration.csproj.
  Restore completed in 199.96 ms for D:\Projects\Others\MassTransit\src\Containers\MassTransit.Containers.Tests\MassTransit.Containers.Tests.csproj.
  Restore completed in 210.4 ms for D:\Projects\Others\MassTransit\src\Loggers\MassTransit.Log4NetIntegration\MassTransit.Log4NetIntegration.csproj.
  Restore completed in 211.78 ms for D:\Projects\Others\MassTransit\src\Containers\MassTransit.UnityIntegration\MassTransit.UnityIntegration.csproj.
  Restore completed in 221.29 ms for D:\Projects\Others\MassTransit\src\MassTransit.AmazonSqsTransport.Tests\MassTransit.AmazonSqsTransport.Tests.csproj.
  Restore completed in 221.55 ms for D:\Projects\Others\MassTransit\src\MassTransit.ActiveMqTransport.Tests\MassTransit.ActiveMqTransport.Tests.csproj.
  Restore completed in 221.54 ms for D:\Projects\Others\MassTransit\src\MassTransit.Azure.ServiceBus.Core\MassTransit.Azure.ServiceBus.Core.csproj.
  Restore completed in 32.28 ms for D:\Projects\Others\MassTransit\src\MassTransit.Futures.Tests\MassTransit.Futures.Tests.csproj.
  Restore completed in 10.45 ms for D:\Projects\Others\MassTransit\src\Persistence\MassTransit.EntityFrameworkIntegration\MassTransit.EntityFrameworkIntegration.csproj.
  Restore completed in 27.22 ms for D:\Projects\Others\MassTransit\src\Persistence\MassTransit.DapperIntegration\MassTransit.DapperIntegration.csproj.
  Restore completed in 34.82 ms for D:\Projects\Others\MassTransit\src\Persistence\MassTransit.DapperIntegration.Tests\MassTransit.DapperIntegration.Tests.csproj.
  Restore completed in 34.97 ms for D:\Projects\Others\MassTransit\src\Persistence\MassTransit.EntityFrameworkIntegration.Tests\MassTransit.EntityFrameworkIntegration.Tests.csproj.
  Restore completed in 46.01 ms for D:\Projects\Others\MassTransit\src\MassTransit.TestFramework\MassTransit.TestFramework.csproj.
  Restore completed in 133.92 ms for D:\Projects\Others\MassTransit\src\MassTransit.HttpTransport.Tests\MassTransit.HttpTransport.Tests.csproj.
  Restore completed in 56.38 ms for D:\Projects\Others\MassTransit\src\MassTransit.Reactive\MassTransit.Reactive.csproj.
  Restore completed in 77.36 ms for D:\Projects\Others\MassTransit\src\Samples\MassTransit.Host.Quartz\MassTransit.Host.Quartz.csproj.
  Restore completed in 62.92 ms for D:\Projects\Others\MassTransit\src\Samples\MassTransit.QuartzService\MassTransit.QuartzService.csproj.
  Restore completed in 55.32 ms for D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj.
  Restore completed in 68.56 ms for D:\Projects\Others\MassTransit\src\MassTransit.RabbitMqTransport\MassTransit.RabbitMqTransport.csproj.
  Restore completed in 161 ms for D:\Projects\Others\MassTransit\src\MassTransit.Host\MassTransit.Host.csproj.
  Restore completed in 39.87 ms for D:\Projects\Others\MassTransit\src\Persistence\MassTransit.NHibernateIntegration\MassTransit.NHibernateIntegration.csproj.
  Restore completed in 47.13 ms for D:\Projects\Others\MassTransit\src\MassTransit.QuartzIntegration\MassTransit.QuartzIntegration.csproj.
  Restore completed in 65.84 ms for D:\Projects\Others\MassTransit\src\MassTransit.SignalR\MassTransit.SignalR.csproj.
  Restore completed in 94.24 ms for D:\Projects\Others\MassTransit\src\Persistence\MassTransit.EntityFrameworkCoreIntegration\MassTransit.EntityFrameworkCoreIntegration.csproj.
  Restore completed in 56.26 ms for D:\Projects\Others\MassTransit\src\Persistence\MassTransit.MartenIntegration\MassTransit.MartenIntegration.csproj.
  Restore completed in 36.38 ms for D:\Projects\Others\MassTransit\src\Persistence\MassTransit.RedisIntegration\MassTransit.RedisIntegration.csproj.
  Restore completed in 51.08 ms for D:\Projects\Others\MassTransit\src\MassTransit.Reactive.Tests\MassTransit.Reactive.Tests.csproj.
  Restore completed in 84.17 ms for D:\Projects\Others\MassTransit\src\MassTransit.HttpTransport\MassTransit.HttpTransport.csproj.
  Restore completed in 87.3 ms for D:\Projects\Others\MassTransit\src\MassTransit.WebJobs.EventHubsIntegration\MassTransit.WebJobs.EventHubsIntegration.csproj.
  Restore completed in 72.91 ms for D:\Projects\Others\MassTransit\src\Persistence\MassTransit.MongoDbIntegration\MassTransit.MongoDbIntegration.csproj.
  Restore completed in 93.4 ms for D:\Projects\Others\MassTransit\src\MassTransit.WebJobs.ServiceBusIntegration\MassTransit.WebJobs.ServiceBusIntegration.csproj.
  Restore completed in 71.04 ms for D:\Projects\Others\MassTransit\src\Persistence\MassTransit.DocumentDbIntegration\MassTransit.DocumentDbIntegration.csproj.
  Restore completed in 73.28 ms for D:\Projects\Others\MassTransit\src\Persistence\MassTransit.DocumentDbIntegration.Tests\MassTransit.DocumentDbIntegration.Tests.csproj.
  Restore completed in 74.2 ms for D:\Projects\Others\MassTransit\src\MassTransit.RabbitMqTransport.Tests\MassTransit.RabbitMqTransport.Tests.csproj.
  Restore completed in 45.63 ms for D:\Projects\Others\MassTransit\src\Persistence\MassTransit.RedisIntegration.Tests\MassTransit.RedisIntegration.Tests.csproj.
  Restore completed in 78.72 ms for D:\Projects\Others\MassTransit\src\MassTransit.QuartzIntegration.Tests\MassTransit.QuartzIntegration.Tests.csproj.
  Restore completed in 46.9 ms for D:\Projects\Others\MassTransit\src\Persistence\MassTransit.MartenIntegration.Tests\MassTransit.MartenIntegration.Tests.csproj.
  Restore completed in 62.64 ms for D:\Projects\Others\MassTransit\src\MassTransit.Transports.Tests\MassTransit.Transports.Tests.csproj.
  Restore completed in 48.55 ms for D:\Projects\Others\MassTransit\src\MassTransit.Tests\MassTransit.Tests.csproj.
  Restore completed in 76.78 ms for D:\Projects\Others\MassTransit\src\MassTransit.SignalR.Tests\MassTransit.SignalR.Tests.csproj.
  Restore completed in 26.01 ms for D:\Projects\Others\MassTransit\src\Samples\Sample.AzureFunctions.ServiceBus\Sample.AzureFunctions.ServiceBus.csproj.
  Restore completed in 54.42 ms for D:\Projects\Others\MassTransit\src\MassTransit.WebJobs.Tests\MassTransit.WebJobs.Tests.csproj.
  Restore completed in 32.28 ms for D:\Projects\Others\MassTransit\src\Persistence\MassTransit.EntityFrameworkCoreIntegration.Tests\MassTransit.EntityFrameworkCoreIntegration.Tests.csproj.
  Restore completed in 54.8 ms for D:\Projects\Others\MassTransit\src\Persistence\MassTransit.MongoDbIntegration.Tests\MassTransit.MongoDbIntegration.Tests.csproj.
  Restore completed in 30.01 ms for D:\Projects\Others\MassTransit\src\Persistence\MassTransit.NHibernateIntegration.Tests\MassTransit.NHibernateIntegration.Tests.csproj.
  Restore completed in 21.72 ms for D:\Projects\Others\MassTransit\src\Samples\MassTransit.SignalR.SampleConsole\MassTransit.SignalR.SampleConsole.csproj.
  Restore completed in 15.53 ms for D:\Projects\Others\MassTransit\src\Samples\RapidTransit.Sample\RapidTransit.Sample.csproj.
  Restore completed in 29 ms for D:\Projects\Others\MassTransit\src\Samples\MassTransit.SignalR.Sample\MassTransit.SignalR.Sample.csproj.

========================================
Build
========================================
Microsoft (R) Build Engine version 16.0.450+ga8dc7f1d34 for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

Context\DictionarySendHeaders.cs(72,40): error CS8652: The feature 'nullable reference types' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\DictionarySendHeaders.cs(72,39): error CS8627: A nullable type parameter must be known to be a value type or non-nullable reference type. Consider adding a 'class', 'struct', or type constraint. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\DictionarySendHeaders.cs(72,10): error CS8652: The feature 'nullable reference types' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\DictionarySendHeaders.cs(72,9): error CS8627: A nullable type parameter must be known to be a value type or non-nullable reference type. Consider adding a 'class', 'struct', or type constraint. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\DictionarySendHeaders.cs(20,18): error CS8646: 'Headers.Get<T>(string, T)' is explicitly implemented more than once. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\DictionarySendHeaders.cs(21,9): error CS0535: 'DictionarySendHeaders' does not implement interface member 'Headers.Get<T>(string, T?)' [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\DictionarySendHeaders.cs(72,20): error CS0111: Type 'DictionarySendHeaders' already defines a member called 'MassTransit.Headers.Get' with the same parameter types [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\JsonHeaders.cs(55,40): error CS8652: The feature 'nullable reference types' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\JsonHeaders.cs(55,39): error CS8627: A nullable type parameter must be known to be a value type or non-nullable reference type. Consider adding a 'class', 'struct', or type constraint. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\JsonHeaders.cs(55,10): error CS8652: The feature 'nullable reference types' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\JsonHeaders.cs(55,9): error CS8627: A nullable type parameter must be known to be a value type or non-nullable reference type. Consider adding a 'class', 'struct', or type constraint. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\JsonHeaders.cs(25,18): error CS8646: 'Headers.Get<T>(string, T)' is explicitly implemented more than once. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\JsonHeaders.cs(26,9): error CS0535: 'JsonHeaders' does not implement interface member 'Headers.Get<T>(string, T?)' [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\JsonHeaders.cs(55,20): error CS0111: Type 'JsonHeaders' already defines a member called 'MassTransit.Headers.Get' with the same parameter types [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Serialization\StaticHeaders.cs(50,40): error CS8652: The feature 'nullable reference types' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Serialization\StaticHeaders.cs(50,39): error CS8627: A nullable type parameter must be known to be a value type or non-nullable reference type. Consider adding a 'class', 'struct', or type constraint. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Serialization\StaticHeaders.cs(50,10): error CS8652: The feature 'nullable reference types' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Serialization\StaticHeaders.cs(50,9): error CS8627: A nullable type parameter must be known to be a value type or non-nullable reference type. Consider adding a 'class', 'struct', or type constraint. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Serialization\StaticHeaders.cs(21,18): error CS8646: 'Headers.Get<T>(string, T)' is explicitly implemented more than once. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Serialization\StaticHeaders.cs(22,9): error CS0535: 'StaticHeaders' does not implement interface member 'Headers.Get<T>(string, T?)' [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Serialization\StaticHeaders.cs(50,20): error CS0111: Type 'StaticHeaders' already defines a member called 'MassTransit.Headers.Get' with the same parameter types [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\DictionarySendHeaders.cs(72,40): error CS8652: The feature 'nullable reference types' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\DictionarySendHeaders.cs(72,39): error CS8627: A nullable type parameter must be known to be a value type or non-nullable reference type. Consider adding a 'class', 'struct', or type constraint. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\DictionarySendHeaders.cs(72,10): error CS8652: The feature 'nullable reference types' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\DictionarySendHeaders.cs(72,9): error CS8627: A nullable type parameter must be known to be a value type or non-nullable reference type. Consider adding a 'class', 'struct', or type constraint. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\DictionarySendHeaders.cs(20,18): error CS8646: 'Headers.Get<T>(string, T)' is explicitly implemented more than once. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\DictionarySendHeaders.cs(21,9): error CS0535: 'DictionarySendHeaders' does not implement interface member 'Headers.Get<T>(string, T?)' [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\DictionarySendHeaders.cs(72,20): error CS0111: Type 'DictionarySendHeaders' already defines a member called 'MassTransit.Headers.Get' with the same parameter types [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\JsonHeaders.cs(55,40): error CS8652: The feature 'nullable reference types' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\JsonHeaders.cs(55,39): error CS8627: A nullable type parameter must be known to be a value type or non-nullable reference type. Consider adding a 'class', 'struct', or type constraint. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\JsonHeaders.cs(55,10): error CS8652: The feature 'nullable reference types' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\JsonHeaders.cs(55,9): error CS8627: A nullable type parameter must be known to be a value type or non-nullable reference type. Consider adding a 'class', 'struct', or type constraint. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\JsonHeaders.cs(25,18): error CS8646: 'Headers.Get<T>(string, T)' is explicitly implemented more than once. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\JsonHeaders.cs(26,9): error CS0535: 'JsonHeaders' does not implement interface member 'Headers.Get<T>(string, T?)' [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\JsonHeaders.cs(55,20): error CS0111: Type 'JsonHeaders' already defines a member called 'MassTransit.Headers.Get' with the same parameter types [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
C:\Program Files\dotnet\sdk\2.2.202\Microsoft.Common.CurrentVersion.targets(1661,5): warning NU1702: ProjectReference 'D:\Projects\Others\MassTransit\src\MassTransit.Futures\MassTransit.Futures.csproj' was resolved using '.NETFramework,Version=v4.6.1' instead of the project target framework '.NETCoreApp,Version=v2.2'. This project may not be fully compatible with your project. [D:\Projects\Others\MassTransit\src\MassTransit.Futures.Tests\MassTransit.Futures.Tests.csproj]

Build FAILED.

C:\Program Files\dotnet\sdk\2.2.202\Microsoft.Common.CurrentVersion.targets(1661,5): warning NU1702: ProjectReference 'D:\Projects\Others\MassTransit\src\MassTransit.Futures\MassTransit.Futures.csproj' was resolved using '.NETFramework,Version=v4.6.1' instead of the project target framework '.NETCoreApp,Version=v2.2'. This project may not be fully compatible with your project. [D:\Projects\Others\MassTransit\src\MassTransit.Futures.Tests\MassTransit.Futures.Tests.csproj]
Context\DictionarySendHeaders.cs(72,40): error CS8652: The feature 'nullable reference types' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\DictionarySendHeaders.cs(72,39): error CS8627: A nullable type parameter must be known to be a value type or non-nullable reference type. Consider adding a 'class', 'struct', or type constraint. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\DictionarySendHeaders.cs(72,10): error CS8652: The feature 'nullable reference types' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\DictionarySendHeaders.cs(72,9): error CS8627: A nullable type parameter must be known to be a value type or non-nullable reference type. Consider adding a 'class', 'struct', or type constraint. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\DictionarySendHeaders.cs(20,18): error CS8646: 'Headers.Get<T>(string, T)' is explicitly implemented more than once. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\DictionarySendHeaders.cs(21,9): error CS0535: 'DictionarySendHeaders' does not implement interface member 'Headers.Get<T>(string, T?)' [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\DictionarySendHeaders.cs(72,20): error CS0111: Type 'DictionarySendHeaders' already defines a member called 'MassTransit.Headers.Get' with the same parameter types [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\JsonHeaders.cs(55,40): error CS8652: The feature 'nullable reference types' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\JsonHeaders.cs(55,39): error CS8627: A nullable type parameter must be known to be a value type or non-nullable reference type. Consider adding a 'class', 'struct', or type constraint. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\JsonHeaders.cs(55,10): error CS8652: The feature 'nullable reference types' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\JsonHeaders.cs(55,9): error CS8627: A nullable type parameter must be known to be a value type or non-nullable reference type. Consider adding a 'class', 'struct', or type constraint. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\JsonHeaders.cs(25,18): error CS8646: 'Headers.Get<T>(string, T)' is explicitly implemented more than once. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\JsonHeaders.cs(26,9): error CS0535: 'JsonHeaders' does not implement interface member 'Headers.Get<T>(string, T?)' [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\JsonHeaders.cs(55,20): error CS0111: Type 'JsonHeaders' already defines a member called 'MassTransit.Headers.Get' with the same parameter types [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Serialization\StaticHeaders.cs(50,40): error CS8652: The feature 'nullable reference types' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Serialization\StaticHeaders.cs(50,39): error CS8627: A nullable type parameter must be known to be a value type or non-nullable reference type. Consider adding a 'class', 'struct', or type constraint. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Serialization\StaticHeaders.cs(50,10): error CS8652: The feature 'nullable reference types' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Serialization\StaticHeaders.cs(50,9): error CS8627: A nullable type parameter must be known to be a value type or non-nullable reference type. Consider adding a 'class', 'struct', or type constraint. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Serialization\StaticHeaders.cs(21,18): error CS8646: 'Headers.Get<T>(string, T)' is explicitly implemented more than once. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Serialization\StaticHeaders.cs(22,9): error CS0535: 'StaticHeaders' does not implement interface member 'Headers.Get<T>(string, T?)' [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Serialization\StaticHeaders.cs(50,20): error CS0111: Type 'StaticHeaders' already defines a member called 'MassTransit.Headers.Get' with the same parameter types [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\DictionarySendHeaders.cs(72,40): error CS8652: The feature 'nullable reference types' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\DictionarySendHeaders.cs(72,39): error CS8627: A nullable type parameter must be known to be a value type or non-nullable reference type. Consider adding a 'class', 'struct', or type constraint. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\DictionarySendHeaders.cs(72,10): error CS8652: The feature 'nullable reference types' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\DictionarySendHeaders.cs(72,9): error CS8627: A nullable type parameter must be known to be a value type or non-nullable reference type. Consider adding a 'class', 'struct', or type constraint. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\DictionarySendHeaders.cs(20,18): error CS8646: 'Headers.Get<T>(string, T)' is explicitly implemented more than once. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\DictionarySendHeaders.cs(21,9): error CS0535: 'DictionarySendHeaders' does not implement interface member 'Headers.Get<T>(string, T?)' [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\DictionarySendHeaders.cs(72,20): error CS0111: Type 'DictionarySendHeaders' already defines a member called 'MassTransit.Headers.Get' with the same parameter types [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\JsonHeaders.cs(55,40): error CS8652: The feature 'nullable reference types' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\JsonHeaders.cs(55,39): error CS8627: A nullable type parameter must be known to be a value type or non-nullable reference type. Consider adding a 'class', 'struct', or type constraint. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\JsonHeaders.cs(55,10): error CS8652: The feature 'nullable reference types' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\JsonHeaders.cs(55,9): error CS8627: A nullable type parameter must be known to be a value type or non-nullable reference type. Consider adding a 'class', 'struct', or type constraint. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\JsonHeaders.cs(25,18): error CS8646: 'Headers.Get<T>(string, T)' is explicitly implemented more than once. [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\JsonHeaders.cs(26,9): error CS0535: 'JsonHeaders' does not implement interface member 'Headers.Get<T>(string, T?)' [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
Context\JsonHeaders.cs(55,20): error CS0111: Type 'JsonHeaders' already defines a member called 'MassTransit.Headers.Get' with the same parameter types [D:\Projects\Others\MassTransit\src\MassTransit\MassTransit.csproj]
    1 Warning(s)
    35 Error(s)

Time Elapsed 00:00:02.04
An error occurred when executing task 'Build'.
Error: One or more errors occurred.
        .NET Core CLI: Process returned an error (exit code 1).
```

### Reproducible Demo

MassTransit *is* the demo.

--- a/README.md
+++ b/README.md
@@ -157,13 +157,19 @@ enhancements and calls for help from people who forget to check back if they get
 
 ## Building from Source
 
- 1. Clone the source down to your machine. 
-   `git clone git://github.com/MassTransit/MassTransit.git`
- 1. Run `build.bat`
+ 1. Install the latest [.NET Core 3.0](https://dotnet.microsoft.com/download/dotnet-core/3.0) preview.
+ 1. Clone the source down to your machine.<br/>
+    ```bash
+    git clone git://github.com/MassTransit/MassTransit.git
+    ```
+ 1. Run `build.ps1` or `build.sh`.
 
 ## Contributing
 
- 1. `git config --global core.autocrlf false`
+ 1. Turn off `autocrlf`.
+    ```bash
+    git config core.autocrlf false
+    ```
  1. Hack!
  1. Make a pull request.
 

